pkg/controller/telemetry/utils.go

```diff
@@ -90,6 +90,34 @@ var (
 		"connection_security_policy",
 	}
 
+	connectionLabels = []string{
```

Comment:
@yp969803 we deliberately not to introduce src dst address into metrics. because in k8s, pods are scaling up/down, even migrating soon, these label are unbounded, could easily lead to memory  leak. similar issues have been reported in istio. You can search there